Formula/glances.rb

```diff
@@ -1,8 +1,8 @@
 class Glances < Formula
   desc "Alternative to top/htop"
   homepage "https://nicolargo.github.io/glances/"
-  url "https://github.com/nicolargo/glances/archive/v3.1.1.tar.gz"
-  sha256 "2fd826d39ed77bcc3656dfff15b4cb3613de9caae0f8e26bd578913110e189fb"
+  url "https://files.pythonhosted.org/packages/99/db/46c306997e4fc220f967716124fe6200ae88f0a82b9ce386886cb720d146/Glances-3.1.2.tar.gz"
```

Comment:
I'm not sure about the release history here but lets keep it to Github for now since they all seem to be available there as well. You would also need to update the list of dependencies assuming some of them also has been upgraded.